linux.md

```diff
@@ -0,0 +1,40 @@
+# .NET for Linux Distributions
+
+.NET can be [installed on Linux](https://learn.microsoft.com/dotnet/core/install/linux) in multiple ways, via packages, [scripts](https://github.com/dotnet/install-scripts), or [tarballs](https://dotnet.microsoft.com/en-us/download). It can be installed globally, locally/privately, or with an app.
+
+.NET is supported on Linux per [.NET Support and Compatibility for Linux Distributions](linux-support.md).
+
+## Official archives
+
+.NET is included in the [official archives](https://pkgs.org/download/dotnet) of the following distributions:
+
+- [Alpine Linux](https://pkgs.alpinelinux.org/packages?name=dotnet*)
+- [Arch Linux](https://archlinux.org/packages/?q=dotnet)
+- [Arch Linux User Repository](https://aur.archlinux.org/packages?K=dotnet)
+- [Fedora](https://packages.fedoraproject.org/search?query=dotnet)
+- [Red Hat Enterprise Linux](https://access.redhat.com/documentation/en-us/net/6.0)
+- [Ubuntu](https://packages.ubuntu.com/search?keywords=dotnet)
+
+In general, you can install the .NET SDK via a versioned package, like `dotnet7` or `dotnet-sdk-7.0`. For example, on Ubuntu 22.10, you can install .NET via the following.
+
+```bash
+sudo apt update && sudo apt install -y dotnet7
```

Comment:
Should we just go with `dotnet-sdk-7.0`? That's the package name that's supported by everyone, including Ubuntu and Microsoft's deb packages?

Reply:
```suggestion
sudo apt update && sudo apt install -y dotnet-sdk-7.0
```